### PR TITLE
add ConversionsParametersStudy

### PR DESCRIPTION
**Pull request by iambrianfung**
production uplift of https://github.com/brave/brave-variations/pull/108

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -830,6 +830,32 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "ConversionsParametersStudy",
```
Comment (tmancey) [inline seed/seed.json:834]: nit: `ConversionsStudy` as we normally just append `Study` to the feature name

Comment (tmancey) [reply seed/seed.json:834]: iOS does not support field trials at this time. We should fix this ASAP on brave-core .

Comment (iambrianfung) [reply seed/seed.json]: @tmancey i carried the name over from what @moritzhaller had originally on `main`.

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` (staging) as per https://github.com/brave/brave-variations/pull/108#issuecomment-921236785.

**Pull request merged**